lib/beaker-pe/pe-client-tools/install_helper.rb

```diff
@@ -0,0 +1,74 @@
+module Beaker
+  module DSL
+    module InstallUtils
+      module PEClientTools
+
+        def install_pe_client_tools_on(hosts, opts = {})
+          # FIXME: accomodate production released location(s)
+          #{{{
+          product = 'pe-client-tools'
+          required_keys = [:puppet_collection, :pe_client_tools_sha, :pe_client_tools_version]
+
+          unless required_keys.all? { |opt| opts.keys.include?(opt) }
+            raise ArgumentError, "The keys #{required_keys.to_s} are required in the opts hash"
+          end
+          urls = { :dev_builds_url   => "http://builds.delivery.puppetlabs.net",
+          }
```

Comment:
This is duplicate code from [here](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/foss_defaults.rb#L16). It's typical usage pattern is [here](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/foss_utils.rb#L1166-L1167). 

The nice thing about using the established pattern here is that if the `dev_builds_url` ever changes, we can react to that without publishing new code, or if you have a location for the installs that's different, this can be changed. If that's something you don't think you'll ever need, that's fine, but the duplicate code should still be eliminated, as the original is accessible from here.